pset-functions.js

```diff
@@ -31,6 +31,31 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+//function blackJack(playerCardScore1,playerCardScore2)
+        //if (playerCardScore1 <= 21 || playerCardScore2 )
+
+
+    var blackJack = function (player1, player2) {
+    if (player1 > 21 && player2 >21)  {
```

Comment:
@deedeemedina great branching logic, the conditions you selected to cover all of the potential outcomes are solid. While your syntax technically works. You generally want to use `if..else if..` in this scenario.  This way the first condition to meet your criteria will be executed and the remaining will be skipped. 

By using individual `if` statements as you've done, every condition will be evaluated every time even if the first one evaluates to true. 

Take a [look a the solution](https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L33) for an example of using `if..else if` to solve this problem